src/plugins/barista/tea.rs

```diff
@@ -0,0 +1,447 @@
+use std::fmt;
+
+use rand::distributions::{Distribution, Standard};
+use rand::seq::SliceRandom;
+use rand::Rng;
+
+const CHANCE_OF_TEA_ADJ: f64 = 0.25;
+const CHANCE_OF_VESSEL_SIZE: f64 = 0.90;
+const CHANCE_OF_VESSEL_ADJ: f64 = 0.85;
+
+// some teas can only be in a certain set of containers. for instance, hohins
```

Comment:
Is this formatting copied from the other impl? Can you capitalize things?

Reply:
Yeah, copied from the other. I cleaned it up.